src/sentry/static/sentry/less/shared-components.less

```diff
@@ -336,14 +336,6 @@ table.table {
     font-weight: 600;
     font-size: 15px;
     margin: 0;
-
-    a {
-      color: @gray-dark;
```

Comment:
Do we not want to update this to use `@blue`?

Reply:
@billyvg these were here to overwrite the default link color value which is set to @blue higher in the stack